### ISSUE TITLE
Import of netapp-cloudmanager_connector_aws fails

### ISSUE THREAD

**Issue by alikulov** [NetApp/terraform-provider-netapp-cloudmanager#55]
Import of netapp-cloudmanager_connector_aws fails.
Terraform versions: v0.14.10, v0.15.5,  v1.0.1.
Provider version: registry.terraform.io/netapp/netapp-cloudmanager v21.6.0

Command I use: `terraform import netapp-cloudmanager_connector_aws.cl-occm-aws i-0339289e5c20097d1`

>  Error: Cannot import non-existent remote object
> 
> While attempting to import an existing object to
> netapp-cloudmanager_connector_aws.cl-occm-aws, the provider detected that no
> object exists with the given id. Only pre-existing objects can be imported;
> check that the id is correct and that it is associated with the provider's
> configured region or endpoint, or use "terraform apply" to create a new remote
> object for this resource.

Way to reproduce:
Create connector via terraform, edit state file and remove a section with created connector, try to import connector.

Terraform code used for connector creation:
```
terraform {
  required_providers {
    netapp-cloudmanager = {
      source = "NetApp/netapp-cloudmanager"
      version = "21.6.0"
    }
  }
}

provider "netapp-cloudmanager" {
  refresh_token = "*******************************"
}


resource "netapp-cloudmanager_connector_aws" "cl-occm-aws" {
  provider      = netapp-cloudmanager
  name          = "netapp-cvo-poc-ConnectorAWS"
  region        = "eu-central-1"
  key_name      = "netapp-cvo-poc"
  company       = "Test Company"
  instance_type = "t3.xlarge"
  aws_tag {
    tag_key   = "Purpose"
    tag_value = "netapp-cvo-poc"
  }
  subnet_id                 = "subnet-09c67948dab3b2654"
  security_group_id         = "sg-0965aa20d2bd2af06"
  iam_instance_profile_name = "netapp-cvo-poc-connectorOCCM1612800196794-OCCMInstanceProfile-1LEQGFWBLJNEO"
  account_id                = ""*******************************""
}
```

State file before removing connector description:
```
{
  "version": 4,
  "terraform_version": "0.14.10",
  "serial": 16,
  "lineage": "dc5bda82-dd44-8beb-f574-8e7b222d7080",
  "outputs": {},
  "resources": [
    {
      "mode": "managed",
      "type": "netapp-cloudmanager_connector_aws",
      "name": "cl-occm-aws",
      "provider": "provider[\"registry.terraform.io/netapp/netapp-cloudmanager\"]",
      "instances": [
        {
          "schema_version": 0,
          "attributes": {
            "account_id": ""*******************************"",
            "ami": null,
            "associate_public_ip_address": true,
            "aws_tag": [
              {
                "tag_key": "Purpose",
                "tag_value": "netapp-cvo-poc12345"
              }
            ],
            "client_id": "A6sLVR1Hzk7uu6GFc8hmfMcYFUjc5LZd",
            "company": "Test Company",
            "enable_termination_protection": false,
            "iam_instance_profile_name": "netapp-cvo-poc-connectorOCCM1612800196794-OCCMInstanceProfile-1LEQGFWBLJNEO",
            "id": "i-0339289e5c20097d1",
            "instance_type": "t3.xlarge",
            "key_name": "netapp-cvo-poc",
            "name": "netapp-cvo-poc-ConnectorAWS",
            "proxy_certificates": null,
            "proxy_password": null,
            "proxy_url": null,
            "proxy_user_name": null,
            "region": "eu-central-1",
            "security_group_id": "sg-0965aa20d2bd2af06",
            "subnet_id": "subnet-09c67948dab3b2654"
          },
          "sensitive_attributes": [],
          "private": ""*******************************""
        }
      ]
    }
  ]
}
```

State file while I'm trying to import connector:
```
{
  "version": 4,
  "terraform_version": "0.14.10",
  "serial": 16,
  "lineage": "dc5bda82-dd44-8beb-f574-8e7b222d7080",
  "outputs": {},
  "resources": [
 
  ]
}

```

**Comment by alikulov** [comment]
13 days passed and no comments. Nice.

**Comment by carchi8py** [comment]
@alikulov we had a Summer shutdown at NetApp, were back today, someone from the team should get back to you later today.

**Comment by wenjun666** [comment]
The issue is when reading the remote objects, the filter with empty attributes are applied to the result. I am looking a workaround to solve the issue.

**Comment by alikulov** [comment]
@wenjun666 Could you please share a rough time-frame for fixing the issue?

**Comment by wenjun666** [comment]
@alikulov This is taking longer than usual due to a API missing. The attribute "company" is not able to retrieve at this point. A user has to manually edit the state file in order to import the resource. That's the best we can do at this point as the conversation with the backend team still going on. Please expect the fix early next week.

**Comment by wenjun666** [comment]
@alikulov It's fixed in version 21.8.1. The ID Terraform uses to import changes to new format: <client_id>:<cconnector_id>
